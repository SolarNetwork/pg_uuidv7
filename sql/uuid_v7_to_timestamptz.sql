SET TIME ZONE UTC;

SELECT uuid_v7_to_timestamptz('018a8920-f523-774b-91ad-a69dde73e64f'::uuid);

SELECT uuid_v7_to_timestamptz('018a8920-f523-774b-91ad-a69dde73e64f'::uuid, 12);
