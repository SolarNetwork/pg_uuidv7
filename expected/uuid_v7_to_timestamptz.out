SET TIME ZONE UTC;
SELECT uuid_v7_to_timestamptz('018a8920-f523-774b-91ad-a69dde73e64f'::uuid);
      uuid_v7_to_timestamptz      
----------------------------------
 Tue Sep 12 11:22:33.123 2023 UTC
(1 row)

SELECT uuid_v7_to_timestamptz('018a8920-f523-774b-91ad-a69dde73e64f'::uuid, 12);
       uuid_v7_to_timestamptz        
-------------------------------------
 Tue Sep 12 11:22:33.123456 2023 UTC
(1 row)

